test/e2e/vmcompose/testdata/TestSetup_127_0_0_1_compose.yaml.golden

```diff
@@ -48,6 +48,7 @@ services:
     container_name: validator01_evm
     image: "ethereum/client-go:latest"
     command:
+      - --config=/geth/config.toml
```

Comment:
not sure we should do command line and file based, I think we should pick one

Reply:
Makes sense, will move everything in file, as staticNodes can't be passed as cmd arg

Reply:
Actually, we have to have a split between config file and args